tests/ui/specs/VirtualList/VirtualGridList/ElementSize/VirtualGridList-GT-28446-specs.js

```diff
@@ -0,0 +1,62 @@
+const Page = require('../VirtualGridListPage');
+
+describe('Change Sapcing', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should change spotlight size when spaceing change [GT-28446]', function () {
+		// Check default item size.
+		const defaultItemWidth = Page.getItemSize().width;
+		const defaultItemHeight = Page.getItemSize().height;
+		const defaultSpacingWidth = Page.itemSpacing(0, 1).width;
+		const defaultSpacingHeight = Page.itemSpacing(0, 5).height;
+		Page.item(0).moveTo();
+		Page.spotlightSelect();
+		const defaultSpotlightWidth = Page.spotlightSize().width;
+		const defaultSpotlightHeight = Page.spotlightSize().height;
+		// Step 3: Knobs > VirtualGridList > spacing > 200
+		Page.inputSpacing.moveTo();
+		Page.spotlightSelect();
+		Page.backSpace();
+		Page.numPad(0);
+		Page.numPad(0);
+		Page.spotlightRight();
+		// Check item size when spacing increase.
+		const newItemWidth = Page.getItemSize().width;
+		const newItemHeight = Page.getItemSize().height;
+		const newSpacingWidth = Page.itemSpacing(0, 1).width;
+		const newSpacingHeight = Page.itemSpacing(0, 4).height;
+		Page.item(0).moveTo();
+		Page.spotlightSelect();
+		const newSpotlightWidth = Page.spotlightSize().width;
+		const newSpotlightHeight = Page.spotlightSize().height;
+		// Step 3-1 Verify: The gap between items grow bigger.
+		expect(newSpacingWidth).to.be.above(defaultSpacingWidth);
+		expect(newSpacingHeight).to.be.above(defaultSpacingHeight);
+		// Step 3-2 Verify: ItemSize is bigger.
+		expect(newItemWidth).to.be.above(defaultItemWidth);
+		expect(newItemHeight).to.be.above(defaultItemHeight);
+		// Step 3-3 Verfiy: SpotlightSize is grow bigger.
+		expect(newSpotlightWidth).to.be.above(defaultSpotlightWidth);
+		expect(newSpotlightHeight).to.be.above(defaultSpotlightHeight);
```

Comment:
I don't think it makes sense to verify the item size. It tends to fit randomly by spacing and total virtualList width. It may not necessarily be above.

Reply:
I agree with you.
Spacing > 0, minHeight is 287 and min Width is 374.
Spacing > 24, minHeight is  280 and minWidth is 364.
Like these cases, depending on the size of the spacing, it can get smaller, bigger, or the same.
I think GT-28446's confirmation is wrong. I don't think we need to compare the item size of this TC, and if we should compare it, need too many cases. It's too much.
Therefore, I would like to change the verify of TC to comparing spacing size only, what do you think? Please let me know.

Reply:
I removed wrong verify for GT-28446. Please check this.